Cargo.toml

```diff
@@ -11,9 +11,14 @@ readme = "README.md"
 homepage = "https://cj.rs/rusqlite_migration"
 rust-version = "1.61"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[features]
+default = []
+async-tokio-rusqlite = ["dep:tokio-rusqlite", "dep:tokio", "dep:tokio-test"]
 
 [dependencies]
+tokio = { version = "1.24.2", features = ["full"], optional = true }
+tokio-rusqlite = { version = "0.3.0", optional = true }
+tokio-test = { version = "0.4.2", optional = true }
```

Comment:
That can be moved to `dev-dependencies`:
```diff
diff --git a/Cargo.toml b/Cargo.toml
index b75b5cb..c3e9e45 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -13,12 +13,11 @@ rust-version = "1.61"
 
 [features]
 default = []
-async-tokio-rusqlite = ["dep:tokio-rusqlite", "dep:tokio", "dep:tokio-test"]
+async-tokio-rusqlite = ["dep:tokio-rusqlite", "dep:tokio"]
 
 [dependencies]
 tokio = { version = "1.24.2", features = ["full"], optional = true }
 tokio-rusqlite = { version = "0.3.0", optional = true }
-tokio-test = { version = "0.4.2", optional = true }
 log = "0.4"
 
 [dependencies.rusqlite]
@@ -27,6 +26,8 @@ default-features = false
 features = []
 
 [dev-dependencies]
 simple-logging = "2.0.2"
 env_logger = "0.9"
 anyhow = "1"
+tokio-test = { version = "0.4.2" }
```

Reply:
Done :).